### ISSUE TITLE
Cannot get /me/drive/root

### ISSUE THREAD

**Issue by jherrera123** [microsoftgraph/msgraph-sdk-dotnet#1876]
**Describe the bug**

Cannot get /me/drive/root using the documented code snippet.

**To Reproduce**
1. Add Microsoft.Graph version 5.8.0 to project file
2. Add code snippet from README.md file to get /me/drive/root information; `var rootItem = await graphClient.Me.Drive.Root.GetAsync();`
3. IDE says: `CS1061: 'DriveRequestBuilder' does not contain a definition for 'Root' and no accessible extension method 'Root' accepting a first argument of type 'DriveRequestBuilder' could be found (are you missing a using directive or an assembly reference?)`

**Expected behavior**
Graph library returns root item

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser N/A
 - Version 11

**Additional context**
I'm trying to upgrade from Graph library 3.x.x to latest version (5.8.0).

**Comment by aroque03** [comment]
Hi folks, we are trying to update to this latest version but ran into the bug above. Any help would be greatly appreciated.

**Comment by andrueastman** [comment]
Thanks for raising this @WolfgangHG 

Any chance this works out for you as the examples in the docs show?

With the guidance in the docs, you would need to fetch the drive id first. 

https://github.com/microsoftgraph/msgraph-sdk-dotnet/blob/dev/docs/upgrade-to-v5.md#drive-item-paths

```cs
 await graphClient.Drives["driveId"].Items["root"].Children.GetAsync();
```

**Comment by estalldecker** [comment]
Hi @andrueastman, I've seen this and similar questions reported in the issues. I'd love to get some context on why with v5 we need to do an extra request to get the `driveId` before working in the `me/drive/root`. In v4 we were able to operate directly on such folder without the need to get the `driveId` first. That extra request might cause some performance impact on latencies for projects with high rps.

**Comment by WolfgangHG** [comment]
Same complaint here: #1848

**Comment by aroque03** [comment]
Agreed. Adding an extra call would affect our load performance. We've invested significantly to expedite our "boot" time which relies on minimizing the number of graph calls.

**Comment by hazzinator1** [comment]
Theres quite a few gaps in the API like this when working with Drives and the Me endpoint (I found the same case when working with Sites as well). I was able to work around it by doing:

`
var driveItem = await this.GraphServiceClient
  .Drives[string.Empty]
  .Items[string.Empty]
  .WithUrl($"https://graph.microsoft.com/v1.0/me/drive/items/{itemId}")
  .GetAsync();
`

It's a bit of a dodgy solution, but can be done in all other cases where you need to access the /me drive, just need to pass in some dummy strings into the indexers to get the correct typings.

**Comment by pfriesch** [comment]
This seems to still be the case. Any progress on this?

**Comment by lvde0** [comment]
Still happening. Seems like a fundamental feature... :/

**Comment by catmanjan** [comment]
Bump, this is soo annoying, it also invalidates a lot of old tutorials
